### PR TITLE
ListToStringConverter seperator as parameter

### PR DESCRIPTION
**Pull request by felixkmh**
Added custom seperator string to ListToString converter using the parameter object

I have verified that:
- [x] These changes work, by building the application and testing them.
- [x] Pull request is targeting `devel` branch.
- [ ] I added myself into [contributors file](https://github.com/JosefNemec/Playnite/blob/devel/source/Playnite.DesktopApp/Resources/contributors.txt) if I want to receive contribution credit in the application itself.

### CODE REVIEW

**Comment by JosefNemec** [comment]
Why? Where is this used? Please create test for this.

**Comment by felixkmh** [comment]
It's just a modification I made to use this Converter in one of my extensions, so I could, for example, use either a newline character or a ", " as a seperator to display list data like platforms, developers, genres, etc., without creating a new Converter for every seperator. And because it does not break the original behaviour either, I thought I might as well push it back up.

File: source/Playnite/Converters/ListToStringConverter.cs [hunk 1]
```diff
@@ -62,7 +64,14 @@ public object Convert(object value, Type targetType, object parameter, System.Gl
                 return string.Empty;
             }
 
-            return string.Join(",", (IEnumerable<object>)value);
+            string sep = defaultSeperator;
+
```
Comment (JosefNemec) [inline source/Playnite/Converters/ListToStringConverter.cs:68]: Remove these empty lines please.

Comment (felixkmh) [reply source/Playnite/Converters/ListToStringConverter.cs]: done

**Pull request merged**